board_client/animations/boing_ball.py

```
@@ -52,32 +52,44 @@ def draw_logo(graphics, t):
-
-    BALL_RADIUS = 2.5
+    # Room bounds scale with display size
+    margin = max(2, min(WIDTH, HEIGHT) // 8)
+    ROOM_X_MIN, ROOM_X_MAX = margin, WIDTH - margin - 1
+    ROOM_Y_MIN, ROOM_Y_MAX = margin, HEIGHT - margin - 1
+    ROOM_Z_MIN, ROOM_Z_MAX = 0, min(WIDTH, HEIGHT) // 2
+
+    # Ball radius and speed scale with display size
+    BASE_RADIUS = max(2, min(WIDTH, HEIGHT) // 7)
+    BALL_RADIUS = BASE_RADIUS
     BALL_COLOR_1 = (0.0, 1.0, 1.0)  # red
```

Ruff (0.11.9)

Local variable `BALL_COLOR_1` is assigned to but never used

Remove assignment to unused variable `BALL_COLOR_1`

(F841)